compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/DeclarationCheckers.kt

```diff
@@ -13,6 +15,7 @@ abstract class DeclarationCheckers {
     open val declarationCheckers: List<FirBasicDeclarationChecker> = emptyList()
     open val memberDeclarationCheckers: List<FirMemberDeclarationChecker> = emptyList()
     open val constructorCheckers: List<FirConstructorChecker> = emptyList()
+    open val controlFlowAnalyserCheckers: List<FirPropertyInitializationChecker> = emptyList()
```

Comment:
Replace type of list to `List<FirControlFlowChecker>`

Reply:
If `FirControlFlowChecker ` will be deleted, should I change type of list to `AbstractFirPropertyInitializationChecker`?